proto-public/pbserverdiscovery/serverdiscovery.proto

```diff
@@ -0,0 +1,33 @@
+// Package serverdiscovery provides a service on Consul servers to discover the set of servers 
+// currently able to handle incoming requests.
+
+syntax = "proto3";
+
+package serverdiscovery;
+
+option go_package = "github.com/hashicorp/consul/proto-public/pbserverdiscovery";
+
+
+service ServerDiscoveryService {
+   rpc WatchServers(WatchServersRequest) returns (stream WatchServersResponse) {};
```

Comment:
As this is a public API, would it be possible to add an explanatory doc comment for this method? The one from the Go implementation is great!

Reply:
Done